lib/materialist.rb

```diff
@@ -1,5 +1,4 @@
 require_relative './configuration'
 
```

Comment:
I thin it's worth require `/version` here too. -- so that at run time we are able to read `Materialist::VERSION`